### ISSUE TITLE
Cookie consent popup documentation enhancement

### ISSUE THREAD

**Issue by anoff** [Lednerb/bilberry-hugo-theme#157]
I am going to implement the [informational cookieconsent](https://cookieconsent.insites.com/app/themes/insites-cookie-consent/examples/example-3-informational.html) banner for my blog. Are you interested in getting this as a PR?

**Comment by Lednerb** [comment]
@anoff Tanks for your request.

If the PR delivers an deactivated by default setting for the `config.toml` I think this would be a nice feature some users would be interested in.

With hugo and a static self hosted blog / website the users should be able to have a privacy friendly website by default / privacy by design.

Because I'm interested:
Which kind of cookies will you use on your site?
Google Analytics tracking? Or do you plan to integrate third-party scripts for ads and so on?

**Comment by anoff** [comment]
I am only using GA atm to get a feeling how my blog posts resonate/which topics get the most traction etc.
Actually I solved the entire thing by using the custom `css/js` feature in the theme. So I don't think inlining the sources into your theme makes much sense given that flexibility.

Should I add a section in the README detailing the few [changes I made](https://github.com/anoff/blog/commit/89cb86331000f7d4d040b7eefe985eaf163eabc2) to get it working?
